mteb/tasks/Retrieval/nld/CQADupstackMathematicaNLRetrieval.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import annotations
+
+import datasets
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class CQADupstackMathematicaNLRetrieval(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="CQADupstackMathematica-NL",
+        description="CQADupStack: A Benchmark Data Set for Community Question-Answering Research. This a "
+        "Dutch-translated version.",
+        reference="https://huggingface.co/datasets/clips/beir-nl-cqadupstack",
+        dataset={
+            "path": "clips/beir-nl-cqadupstack",
+            "revision": "1de13232f452f0b4f5525d8c6c41cd6a9ae1d084",
+            "split": "mathematica",
+        },
+        type="Retrieval",
+        category="s2p",
+        modalities=["text"],
+        eval_splits=["test"],
+        eval_langs=["nld-Latn"],
+        main_score="ndcg_at_10",
+        date=("2024-10-01", "2024-10-01"),
+        domains=["Written"],
```

Comment:
Seems like the domains are only minimally filled out. It seems like at least "Non-fiction" would apply to many of these.